bin/lamassu-update

```diff
@@ -71,6 +71,15 @@ BACKUP_CRON="@daily $BACKUP_CMD > /dev/null"
 ( (crontab -l 2>/dev/null || echo -n "") | grep -v '@daily.*lamassu-backup-pg'; echo $BACKUP_CRON ) | crontab - >> $LOG_FILE 2>&1
 $BACKUP_CMD >> $LOG_FILE 2>&1
 
+decho "updating motd scripts
+set +e
+chmod -x /etc/update-motd.d/*-release-upgrade     
```

Comment:
Why chmod instead of just deleting? Also, do we want to add our own motd?

Reply:
No reason, though perhaps we want to re-enable update/release notifications if we have future versions that are update-resilient.

We could add our own motd, one that displays current and latest server versions, server process status, etc.

However, would prefer to do this for now and circle back to that, since we need to pin down lamassu-install for #stable soon.